core/domain/prod_validators_test.py

```diff
@@ -9231,20 +8916,11 @@ def test_model_validation_fails_for_negative_translation_reviewer_counts(
             self.sample_language_code: self.negative_count}
         stats_model.put()
         expected_output = [
-            u'[u\'failed validation check for translation reviewer count check '
-            'of CommunityContributionStatsModel\', [u\'Entity id %s: '
-            'Translation reviewer count for language code %s: %s does not '
-            'match the expected translation reviewer count for language code '
-            '%s: 0\']]' % (
-                stats_model.id, self.sample_language_code,
-                stats_model.translation_reviewer_counts_by_lang_code[
-                    self.sample_language_code], self.sample_language_code),
-
             u'[u\'failed validation check for domain object check of '
             'CommunityContributionStatsModel\', [u\'Entity id %s: Entity '
             'fails domain validation with the error Expected the translation '
             'reviewer count to be non-negative for %s language code, '
-            'received: %s.\']]' % (
+            'recieved: %s.\']]' % (
```

Comment:
Do you remember making this change here from "received" to "recieved"?

Reply:
No; I don't remember changing that. I am pretty sure only cut, copied, and pasted.